datajunction-clients/python/README.md

```diff
@@ -188,6 +188,28 @@ djbuilder.restore_namespace("foo")
 The `cascade` parameter in both of above methods allows for cascading
 effect applied to all underlying nodes and namespaces. Use it with caution!
 
+### Tags
+
+You can read existing tags as well as create new ones.
+```python
+djbuilder.tag(name="deprecated", description="This node has been deprecated.", tag_type="standard", tag_metadata={"contact": "Foo Bar"})
+
+Tag(dj_client=..., name='deprecated', description='This node has been deprecated.', tag_type='standard', tag_metadata={"contact": "Foo Bar"})
+```
+```python
+djbuilder.tag("official")
+
+[DJClientException]: Tag `official` does not exist.
+```
+
+To create a tag:
```

Comment:
Maybe add an example of how to add/update tag on a node?

Reply:
Good idea, updated!